.npmignore.publishr

```diff
@@ -0,0 +1,9 @@
+/*
+!/dist
+dist/*.map
+!/lib
+!/src
+!LICENSE.txt
+!CHANGELOG.md
+!README.md
+!package.json
```

Comment:
This LGTM. Chiming in on the earlier Slack discussion, I think that it is better to go with a whitelist like this. I also thing that being explicit (by keeping in stuff like !package.json and !README.md) is a good way to go.